src/zenml/zen_server/deploy/terraform/recipes/gcp/zen_server.tf

```diff
@@ -47,7 +47,7 @@ resource "helm_release" "zen-server" {
   }
   set {
     name = "ingress.host"
-    value = var.create_ingress_controller? "${data.kubernetes_service.ingress-controller[0].status.0.load_balancer.0.ingress.0.ip}.nip.io" : var.ingress_controller_hostname
+    value = var.create_ingress_controller? "${data.kubernetes_service.ingress-controller[0].status.0.load_balancer.0.ingress.0.ip}.nip.io" : "${var.ingress_controller_hostname}.nip.io"
```

Comment:
I didn't notice this before, but thought I should mention it now: .nip.io only works with IP addresses, but you're using the ingress hostname here, which may or may not be an IP address.

Reply:
You're right but I had noticed that in the case of GKE and AKS, the ingress controller host is an IP address and so I have added this suffix only to those two recipes.